### PR TITLE
Docs for getitem/loc/iloc/at/iat

### PR DESCRIPTION
**Pull request by 1e-to**

### CODE REVIEW

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 1]
```diff
@@ -62,22 +62,20 @@
...
+    for methods `pandas.Series.iloc`, `pandas.Series.loc`, `pandas.Series.iat`, `pandas.Series.at`
 
     Parameters
     ----------
-    series: :obj:`pandas.Series`
-           input series
-    idx: :obj:`int`, :obj:`slice` or :obj:`pandas.Series`
-        input index
+    self: :class:`pandas.Series`
+        input arg
```
Comment (akharche) [inline sdc/datatypes/hpat_pandas_series_functions.py]: input Series?

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 2]
```diff
@@ -174,47 +172,84 @@ def hpat_pandas_series_getitem(self, idx):
...
+    Test: python -m sdc.runtests -k sdc.tests.test_series.TestSeries.test_getitem_series*
 
     Parameters
     ----------
-    series: :obj:`pandas.Series`
-        input series
-    idx: :obj:`int`, :obj:`slice` or :obj:`pandas.Series`
-        input index
+    self: :class:`pandas.Series`
+        input arg
```
Comment (akharche) [inline sdc/datatypes/hpat_pandas_series_functions.py]: same

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 3]
```diff
@@ -442,6 +515,65 @@ def hpat_pandas_series_iloc_impl(self):
...
+    .. literalinclude:: ../../../examples/series_loc/series_loc_slice.py
+       :language: python
+       :lines: 27-
+       :caption: With a slice object. Returns multiple value.
+       :name: ex_series_loc
+
+    .. command-output:: python ./series_loc/series_loc_slice.py
+       :cwd: ../../../examples
+
+    .. note::
```
Comment (AlexanderKalistratov) [inline sdc/datatypes/hpat_pandas_series_functions.py]: I think there should be limitation block.
```
    Limitations
    -----------
```
Or something like that.
@PokhodenkoSA ?

**Comment by AlexanderKalistratov** [comment]
@akharche are you fine with this PR?

**Comment by AlexanderKalistratov** [comment]
@akharche ping

**Pull request merged**